src/main/java/com/patchworkmc/mapping/MemberMap.java

```diff
@@ -0,0 +1,32 @@
+package com.patchworkmc.mapping;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MemberMap {
```

Comment:
It's not immediately obvious what this class is holding and why. Could some docs be added?

Reply:
It's a map of a member to an arbitrary string, good idea.